governance/constitution.md

```diff
@@ -0,0 +1,34 @@
+# Article I
+No user of this blockchain shall make knowingly false or misleading statements, nor statements that constitute discrimination or harassment, nor profit thereby.
+# Article II
+The rights of contract and of private property shall be inviolable, therefore no property shall change hands except with the consent of the owner or by a lawful Arbitrator’s order.
+# Article III
+Each Member agrees to resolve disputes through the blockchain’s default arbitration process, or any other process that the parties to a transaction may mutually agree to in advance.
+# Article IV
+No Member shall offer nor accept anything of value in exchange for a vote of any type, including for Block Producer candidates, Amendments or Worker Proposals, nor shall any Member unduly influence the vote of another.
+# Article V
+This EOS Blockchain has no owner, manager or fiduciary. It is governed exclusively under the terms of this Constitution.
+# Article VI
+No Member nor any Beneficial Interest shall own more than 10% of issued tokens.
+# Article VII
+Each Member agrees that penalties for violations may include, but are not limited to, fines, account freezing, and reversal of transactions.
+# Article VIII
+No Member shall serve as a Block Producer who has not agreed in advance to the Block Producer Agreement provided by the Members of this blockchain.
+# Article IX
+No Arbitrator shall serve except within an Arbitration Forum.
+# Article X
+Sets minimum standards for an Arbitrator and an Arbitration Forum.
```

Comment:
This appears to be a title rather than the contents of an article.

Reply:
Correct. Fixing that now.